### PR TITLE
CSO-31-Roles updated with add reviewer/admin

### PR DESCRIPTION
**Pull request by zachlawton**

### CODE REVIEW

File: app/controllers/syllabi_controller.rb [hunk 1]
```diff
@@ -1,13 +1,46 @@
...
-  
\ No newline at end of file
+    # def admin
+    #   redirect_to root_path if current_user.role == 1
+    #   redirect_to syllabi_reviewer_url if current_user.role == 2
+    #   @admins = Admin.where(user_id: current_user.id)
+    #   @admin_found = Admin.find_by_id(current_user.id)
+    # end
+
+    # def reviewer
```
Comment (kaigomes7) [inline app/controllers/syllabi_controller.rb:24]: Will the reviewer redirect still work if this is commented out?

File: app/views/admins/edit.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<%= link_to("Back to Admin List", admins_path, :class => 'back-link') %>
+<!-- <%= link_to("Back to Admin List", admins_path, :class => 'back-link') %>
```
Comment (kaigomes7) [inline app/views/admins/edit.html.erb:1]: I think this is messing up part of brakeman, the full output is below. Note warnings will also cause brakeman to fail so I recommend just ignoring them, there's an easy way to do it but I forgot, should be easy to Google.

== Errors ==

Error: app/views/user/index.html.erb:32 :: parse error on value "<" (tLT) Could not parse app/views/user/index.html.erb
Location: Could not parse /csce431/cibs-syllabus-organizer/app/views/user/index.html.erb

== Warnings ==

Confidence: Medium
Category: Mass Assignment
Check: PermitAttributes
Message: Potentially dangerous key allowed for mass assignment
Code: params.require(:user).permit(:name, :email, :role)
File: app/controllers/user_controller.rb
Line: 47

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -15,14 +15,18 @@
         <% end %>
```
Comment (kaigomes7) [inline app/views/shared/_navbar.html.erb:15]: Make sure to update with main, I think a few things on the navbar have been updated

**Review by kaigomes7 — COMMENTED** [review]
Looks great, hopefully brakeman can be fixed

File: app/controllers/syllabi_controller.rb (hunk 1 above)
Comment (LanceAbuan) [reply app/controllers/syllabi_controller.rb:24]: It should. Look at line 7-10, I think that is what replaces what you were looking at.

**Review by kaigomes7 — DISMISSED** [review]
LGTM

**Review by kaigomes7 — APPROVED** [review]

**Pull request merged**